### PR TITLE
Add troubleshooting hint to logging

### PR DESCRIPTION
**Pull request by raboof**
Triggered by https://github.com/akka/akka-management/issues/306

### CODE REVIEW

File: akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardCoordinator.scala [hunk 1]
```diff
@@ -1025,7 +1025,8 @@ class DDataShardCoordinator(typeName: String, settings: ClusterShardingSettings,
 
     case UpdateTimeout(CoordinatorStateKey, Some(`evt`)) ⇒
       log.error(
-        "The ShardCoordinator was unable to update a distributed state within 'updating-state-timeout': {} millis (retrying), event={}",
+        "The ShardCoordinator was unable to update a distributed state within 'updating-state-timeout': {} millis (retrying). " +
+          "Perhaps the ShardRegion has not started on all active nodes yet? event={}",
```
Comment (patriknw) [inline akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardCoordinator.scala]: toString of these events should be ok

**Review by patriknw — APPROVED** [review]
LGTM

**Review by chbatey — APPROVED** [review]
:+1: this has come up a few times

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**